setup.cfg

```diff
@@ -60,6 +60,8 @@ test =
     pytest !=5.3.4
     pytest-cov
     pytest-doctestplus
+zstd =
```

Comment:
Can you add `%(zstd)s` to the `all` extra? That will ensure this code gets tested.

Reply:
Done.